src/sonic-bgpcfgd/bgpcfgd/main.py

```diff
@@ -53,6 +54,8 @@ def do_work():
         BGPAllowListMgr(common_objs, "CONFIG_DB", "BGP_ALLOWED_PREFIXES"),
         # BBR Manager
         BBRMgr(common_objs, "CONFIG_DB", "BGP_BBR"),
+        # Static Route Managers
+        StaticRouteMgr(common_objs, "CONFIG_DB", "STATIC_ROUTE"),
     ]
     runner = Runner(common_objs['cfg_mgr'])
     for mgr in managers:
```

Comment:
I dont see any code in "RouteOrch::addRoute" to trigger the proactive nbr resolution when nexthop is not resolved, is this already planned? Did we test fast path routing with static route config in the HW?

Reply:
Resolving nexthop in RouteOrch is our next step for the static route tasks.

Reply:
ok